common/src/main/scala/common/collections/EnhancedCollections.scala

```diff
@@ -16,7 +15,7 @@ object EnhancedCollections {
     * After trying and failing to do this myself, I got this to work by copying the answer from here:
     * https://stackoverflow.com/questions/29886246/scala-filter-by-type
     */
-  implicit class EnhancedTraversableLike[T2, Repr <: TraversableLike[T2, Repr], That](val traversable: TraversableLike[T2, Repr]) extends AnyVal {
+  implicit class EnhancedIterableOps[T2, Repr[x] <: IterableOps[x, Repr, Repr[x]]](val iterableOps: IterableOps[T2, Repr, Repr[T2]]) extends AnyVal {
```

Comment:
Since the entire `Traversable` family is gone in 2.13 this is now done with `IterableOps`. The changes I made don't appear to be exactly the same as what was here before; a few of the ascriptions that were applied for callers of this code in the baseline are not only unnecessary but do not work with these changes.